static/src/javascripts/projects/common/modules/experiments/tests/contributions-epic-brexit-supreme.js

```diff
@@ -39,7 +39,7 @@ define([
         this.author = 'Phil Wills';
         this.description = 'Appeal linked to the Brexit appeal in the Supreme Court';
         this.showForSensitive = false;
-        this.audience = 1;
+        this.audience = 0.7;
```

Comment:
Was set to `0.7` to facilitate further testing. However, if we no longer need to run additional tests, then I agree with changing it to `0.88` (along with the offset for the always ask strategy).